code/game/objects/items/devices/radio/headset.dm

```diff
@@ -328,20 +328,24 @@ GLOBAL_LIST_INIT(channel_tokens, list(
 	worn_icon_state = "cent_headset"
 	keyslot = /obj/item/encryptionkey/headset_cent
 	keyslot2 = /obj/item/encryptionkey/headset_com
+	command = TRUE
```

Comment:
Could remove `command = true` from all of its children for a bit of less code <details><summary>Paths</summary>`obj/item/radio/headset_cent/empty` | `/obj/item/radio/headset_cent/commander` | `/obj/item/radio/headset_cent/alt`</details> children take from the parent variables

Reply:
Sure thing - and voila!